edb/server/args.py

```diff
@@ -463,6 +464,10 @@ def _writer(status: str) -> None:
              'automatically scale up (to --compiler-pool-size workers ) and '
              'down to the demand. Default to "fixed".',
     ),
+    click.option(
+        '--compiler-pool-addr',
```

Comment:
I'm not sure a Unix socket for the remote pool is worth it operationally.  It's a lot easier to operate with TCP in prod.

Reply:
to clarify: I'm OK if we support Unix too, it's just that it shouldn't be the only mode

Reply:
On a second thought, let's not make our lives harder and support TCP only.